android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/surveydetail/SurveyDetailScreen.kt

```diff
@@ -0,0 +1,127 @@
+package co.nimblehq.ic.kmm.suv.android.ui.screens.surveydetail
+
+import android.annotation.SuppressLint
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.material.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import co.nimblehq.ic.kmm.suv.android.ui.components.BackButton
+import co.nimblehq.ic.kmm.suv.android.ui.theme.Typography
+import coil.compose.AsyncImage
+
+@Composable
+fun SurveyDetailScreen(
+    viewModel: SurveyDetailViewModel,
+    onBackClick: () -> Unit = {}
+) {
+    SurveyDetailScreenContent(
+        viewModel.contentUiModel,
+        onBackClick
+    )
+}
+
+data class SurveyDetailContentUiModel(
+    val title: String,
+    val description: String,
+    val imageUrl: String
+)
+
+@SuppressLint("UnusedMaterialScaffoldPaddingParameter")
+@Composable
+private fun SurveyDetailScreenContent(
+    uiModel: SurveyDetailContentUiModel,
+    onBackClick: () -> Unit = {}
+) {
+    Box(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(Color.Black)
+    ) {
+        Box(modifier = Modifier.fillMaxSize()) {
+            AsyncImage(
+                model = uiModel.imageUrl,
+                contentDescription = null,
+                contentScale = ContentScale.FillHeight,
+                modifier = Modifier.matchParentSize()
+            )
+            Box(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .background(
+                        brush = Brush.verticalGradient(
+                            colors = listOf(
+                                Color.Transparent,
+                                Color.Black.copy(alpha = 0.6f)
+                            )
+                        )
+                    )
+            )
+        }
+    }
+    Scaffold(
+        topBar = {
+            TopAppBar(
+                backgroundColor = Color.Transparent,
+                elevation = 0.dp,
+                modifier = Modifier
+                    .statusBarsPadding()
+
+            ) {
+                BackButton(onBackClick)
+            }
+        },
+        drawerBackgroundColor = Color.Transparent,
+        contentColor = Color.Transparent,
+        backgroundColor = Color.Transparent
+    ) { _ ->
+        Column(
```

Comment:
Can we get rid of `_ ->`? 

```suggestion
        backgroundColor = Color.Transparent
    ) {
        Column(
```

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/112/commits/2510eab5048f645017fb6c8e5dd19f801b627b91